USER_GUIDE.md

```diff
@@ -214,6 +214,53 @@ DeleteIndexRequest deleteIndexRequest = new DeleteRequest.Builder().index(index)
 DeleteIndexResponse deleteIndexResponse = client.indices().delete(deleteIndexRequest);
 ```
 
+## Data Stream API
```

Comment:
Sorry one more comment: can you add the new section to TOC as well?

Reply:
Added